lib/renderer/content-scripts-injector.js

```diff
@@ -1,6 +1,8 @@
-const {ipcRenderer} = require('electron')
+const {ipcRenderer, webFrame} = require('electron')
 const {runInThisContext} = require('vm')
 
+webFrame.registerURLSchemeAsPrivileged('chrome-extension')
```

Comment:
I think determining whether `chrome-extension://` can be considered as secure is independent of whether we injected content-scripts or not.
That being said, for now, I see no problem with both solutions.